src/harness/certs/generate_short_lived_certs.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash -e
+
+# Setup: build intermediate directories.
+mkdir -p root
+touch index.txt
+echo -n 'unique_subject = no' >> index.txt.attr
+
+function generate_cbsd_short_lived_certificate()
+{
+  # Argument1: Name of the certificate to be generated.
+  # Argument2: Time (in minutes).
+
+  # Create certificate for test case which use short lived certificate.
+  if [ -f cbsd_ca.cert -a -f private/cbsd_ca.key ]; then
+    echo "Create short lived certificate"
+    current_time=`date -u +%y%m%d%H%M%SZ`
+    offset=$2
+    enddate_value=$(date -u -d "now + $offset minutes" '+%y%m%d%H%M%SZ')
+
+    # Generate normal operation short_lived certificate/key.
+    echo "\n\nGenerate 'short_lived' certificate/key"
```

Comment:
This needs `echo -e` to work correctly:

    $ echo "\ntest"
    \ntest
    $ echo -e "\ntest"
    
    test

Reply:
fixed both places. Thanks for the findings.